main.js

```diff
@@ -83,6 +83,15 @@ Trello.prototype.addListToBoard = function (boardId, name, callback) {
     return makeRequest(rest.post, this.uri + '/1/boards/' + boardId + '/lists', {query: query}, callback);
 };
 
+Trello.prototype.addMemberToBoard = function (boardId, memberId, type, callback) {
+    var query = this.createQuery();
+    var data = {};
+
+    data.type = type; // Valid Values: 'normal','admin','observer'
+
+    return makeRequest(rest.put, this.uri + '/1/boards/' + boardId + '/members/' + memberId,{ data: data, query: query }, callback);
```

Comment:
Sorry, I'm being really nitpicky, but you've got a space after every single comma _except_ the one at `memberId,` 😄 wanna add a space for consistency?